packages/Takeover/src/components/Header/Header.styles.js

```diff
@@ -28,10 +28,36 @@ export const Header = styled.div(
 );
 
 export const Heading = styled(OriginalHeading)`
-  margin: 0 0 0 ${spacer(2)};
+  margin: 0;
+  padding: 18px 0 17px ${spacer(3)};
```

Comment:
```suggestion
  padding: ${spacer(2)} 0 ${spacer(2)} ${spacer(3)};
```

Reply:
sure

Reply:
@allison-c @jonathanjakimon isn't too concerned about the padding as long as the header is centered and the height is at 64px. So I remove the top and bottom and let ` align-items: center;` to do the work